internal/xds/translator/listener.go

```diff
@@ -785,3 +790,55 @@ func buildHeadersWithUnderscoresAction(in *ir.HeaderSettings) corev3.HttpProtoco
 	}
 	return corev3.HttpProtocolOptions_REJECT_REQUEST
 }
+
+func buildForwardClientCertDetailsAction(in *ir.HeaderSettings) hcmv3.HttpConnectionManager_ForwardClientCertDetails {
+	if in != nil {
+		if in.ForwardClientCert != nil {
```

Comment:
can we merge these conditions? just to simplify the code

Reply:
I don't think we can merge this, there is possibility that headers is nil.